### PR TITLE
Mappers

### PR DESCRIPTION
**Pull request by injacmarko**

### CODE REVIEW

**Review by KoAleksandar — APPROVED** [review]

File: src/BuisinessLogic/Converter.cs [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-    public class Converter
+    public static class Converter
     {
-        public double KmToMi(double Km)
+        public static double KmToMi(this double Km)
         {
             return Km / 1.609;
         }
-        public double MiToKm(double Mi)
+        public static double MiToKm(this double Mi)
```
Comment (djordjevics) [inline src/BuisinessLogic/Converter.cs]: getter in class directly

Comment (djordjevics) [reply src/BuisinessLogic/Converter.cs]: count for all methodss

File: src/BuisinessLogic/Mappers/APIToBusiness.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+﻿using BusinessLogic.Models.Enums;
+using BusinessLogic.Models;
+
+namespace BusinessLogic.Mappers
+{
+    public static class APIToBusiness
+    {
+        public static VehicleDTO toVehicleDTO(this CreateVehicle create)
```
Comment (djordjevics) [inline src/BuisinessLogic/Mappers/APIToBusiness.cs]: naming..

create -> request or just vehicle

File: src/BuisinessLogic/Mappers/APIToBusiness.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            {
+                Color = req.UpdatedVehicle.Color,
+                Model = req.UpdatedVehicle.Model,
+                LicensePlate = req.UpdatedVehicle.LicensePlate,
+                WheelCount = req.UpdatedVehicle.WheelCount,
+                FuelType = req.UpdatedVehicle.FuelType
+            };
+        }
+
+        public static FuelLogDTO toFuelLogDTO(this CreateFuelLog create)
```
Comment (djordjevics) [inline src/BuisinessLogic/Mappers/APIToBusiness.cs]: same, naming

File: src/BuisinessLogic/Mappers/BusinessToData.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
+﻿using BusinessLogic.Models.Enums;
+using BusinessLogic.Models;
+using Data.Models;
+
+namespace BusinessLogic.Mappers
+{
+    public static class BusinessToData
+    {
+
+        public static FuelLogData toFuellogData(this FuelLogDTO dto)
```
Comment (djordjevics) [inline src/BuisinessLogic/Mappers/BusinessToData.cs]: can be simplified

File: src/BuisinessLogic/Mappers/DataToBusiness.cs [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        public static VehicleDTO toVehicleDTO(this VehicleData data)
+        {
+            return new VehicleDTO
+            {
+                Id = data.Id,
+                Model = data.Model,
+                Color = data.Color,
+                LicensePlate = data.LicensePlate,
+                WheelCount = data.WheelCount,
+                FuelType = (FuelType)Enum.Parse(typeof(FuelType), data.FuelType)
```
Comment (djordjevics) [inline src/BuisinessLogic/Mappers/DataToBusiness.cs]: Parse<FuelType>(out enumValue) ?

**Review by djordjevics — CHANGES_REQUESTED** [review]

**Review by FilipZecevic — APPROVED** [review]

**Review by FilipZecevic — APPROVED** [review]

**Review by djordjevics — APPROVED** [review]

**Pull request merged**